lua/nvim-dap-virtual-text/virtual_text.lua

```diff
@@ -94,27 +94,38 @@ function M.set_virtual_text(stackframe)
   end
 
   for line, content in pairs(virtual_text) do
+    if vim.g.dap_virtual_text_commented then
+      content = vim.fn.substitute(vim.o.commentstring, "%s", content, nil)
```

Comment:
`vim.o.commentstring:gsub('%%s', content)` would be a more Lua solution but I'm fine with this version.